.travis.yml

```diff
@@ -12,39 +16,43 @@ install:
 
 matrix:
   include:
+    - stage: linting
+      python: 3.7
+      env: TOX_ENV="py37-linting"
+
     - stage: test
       python: 3.5
-      env: TOX_ENV="py35-nameko{2.11,2.12,latest}-redis2.10"
+      env: TOX_ENV="py35-nameko{2.11,2.12,latest}-redis2.10-test"
```

Comment:
I wondered if it might be possible to simplify this config using [tox-travis](https://tox-travis.readthedocs.io/), but no need to do that here, as you've got it all working.

Let's create an issue for that and I'll have a go. Assuming you're sticking with travis for the moment?

Reply:
Yes, we're sticking with Travis CI for the time being.
It sounds good, please feel free to create an issue for that.